static/js/containers/LoginButton.js

```diff
@@ -7,9 +7,23 @@ class LoginButton extends React.Component {
   render() {
     const { authentication } = this.props;
 
-    return <DropdownButton title={authentication.name}>
-      <MenuItem eventKey="logout">Logout</MenuItem>
-    </DropdownButton>;
+    return (
+      <DropdownButton
+        title={authentication.name}
+        id="logout-button">
+        {this.logOutButton()}
```

Comment:
I'm not sure if it's worth making it a separate function here. Would it be possible to inline it?